src/components/AnchorForAttachmentsOnly/BaseAnchorForAttachmentsOnly.tsx

```diff
@@ -10,57 +9,51 @@ import * as ReportUtils from '@libs/ReportUtils';
 import * as Download from '@userActions/Download';
 import CONST from '@src/CONST';
 import ONYXKEYS from '@src/ONYXKEYS';
-import {defaultProps as anchorForAttachmentsOnlyDefaultProps, propTypes as anchorForAttachmentsOnlyPropTypes} from './anchorForAttachmentsOnlyPropTypes';
-
-const propTypes = {
-    /** Press in handler for the link */
-    onPressIn: PropTypes.func,
-
-    /** Press out handler for the link */
-    onPressOut: PropTypes.func,
+import {Download as OnyxDownload} from '@src/types/onyx';
+import AnchorForAttachmentsOnlyProps from './AnchorForAttachmentsOnlyTypes';
 
+type BaseAnchorForAttachmentsOnlyPropsWithOnyx = {
     /** If a file download is happening */
-    download: PropTypes.shape({
-        isDownloading: PropTypes.bool.isRequired,
-    }),
-
-    ...anchorForAttachmentsOnlyPropTypes,
+    download: OnyxEntry<OnyxDownload>;
 };
 
-const defaultProps = {
-    onPressIn: undefined,
-    onPressOut: undefined,
-    download: {isDownloading: false},
-    ...anchorForAttachmentsOnlyDefaultProps,
-};
+type BaseAnchorForAttachmentsOnlyProps = AnchorForAttachmentsOnlyProps &
+    BaseAnchorForAttachmentsOnlyPropsWithOnyx & {
+        /** Press in handler for the link */
+        onPressIn?: () => void;
+        /** Press out handler for the link */
+        onPressOut?: () => void;
+    };
 
-function BaseAnchorForAttachmentsOnly(props) {
-    const sourceURL = props.source;
+function BaseAnchorForAttachmentsOnly({style, source, displayName, download, onPressIn, onPressOut}: BaseAnchorForAttachmentsOnlyProps) {
```

Comment:
```suggestion
function BaseAnchorForAttachmentsOnly({style, source = '', displayName = '', download, onPressIn, onPressOut}: BaseAnchorForAttachmentsOnlyProps) {
```

According to the deleted `src/components/AnchorForAttachmentsOnly/anchorForAttachmentsOnlyPropTypes.js` file.